packages/good-design/src/core/web3/modals/TxModal.tsx

```diff
@@ -34,6 +39,35 @@ const txModalCopy = {
 
 const TxModalContent = ({ content }: { content: string }) => <TransText t={content} variant="sm-grey-650" />;
 
+const TxContentMultiClaim = ({ content }: { content: string }) => (
+  <VStack space={2} paddingX={2}>
+    <TransText t={content} variant="sm-grey-650" />
+    <TransText
+      t={/*i18n*/ "It may take over a minute for transaction(s) to be signed."}
```

Comment:
a minute?

Reply:
the faucet hook still relies on useEtherBalance, and refresh-rate is set to 12 blocks.

should this hook be refactored?

Reply:
i already explained refresh of faucet happens when new tx happens

Reply:
So what is the request to change here?

Reply:
also, additionally:

Refresh of faucet does not happen when doing a tx.
it takes around 12 blocks for it receive the updated balance

![image](https://github.com/user-attachments/assets/675bc19b-31f1-4c76-b541-8cfb2c13cdb6)

Reply:
the faucet code has been fixed on branch 116, check time now

Reply:
is instant, so the fix worked